src/components/Display.jsx

```diff
@@ -0,0 +1,12 @@
+/* eslint-disable quotes */
+import React from "react";
+
+function Display() {
+  return (
+    <div id="display" className="display">
+      150
+    </div>
+  );
+}
+
```

Comment:
- [ ] The Display component should accept the "result" prop and should be checked with prop-types with a value of "string". 
- [ ] It should also have a default prop with a value of "0".